DigitalExcellence/dex-frontend
# Feature/597 tags first version

## Description

Added tags to DeX! 
In this first version of tags I completed making functionallity to add  a project with tags, display the tags where needed and edit the tags on an existing project. 

## Type of change


## Checklist


## Steps to Test or Reproduce

Check that the tags work well and are complient with the designs in the issue 

1. Add a project with tags
2. Go to project overview to see tags there
3. Go to project detail to view tags there
3. Try and edit the tags on the project

##597

Closes: #597
